spec/controllers/tasks_controller_spec.rb

```diff
@@ -154,6 +155,11 @@
         expect(assigns(:task).license).to match(valid_params[:license_id])
       end
 
+      it 'sets labels correctly' do
+        post_request
+        expect(assigns(:task).labels).to eq(valid_params[:label_ids])
```

Comment:
the expectation would have to be updated accordingly

I would also like a spec to check if something has been saved into the database (ie check if TaskLabel count increases)

Reply:
I added the test for the TaskLabel count.